RecoLocalCalo/HcalRecAlgos/src/PedestalSub.cc

```diff
@@ -0,0 +1,108 @@
+#include <iostream>
+#include <cmath>
+#include <climits>
+#include "RecoLocalCalo/HcalRecAlgos/interface/PedestalSub.h"
+
+using namespace std;
+
+PedestalSub::PedestalSub() {
+}
+
+PedestalSub::~PedestalSub() { 
+}
+
+void PedestalSub::Init(Method method=AvgWithThresh, int runCond=0, float threshold=0.0, float quantile=0.0) {
+  fMethod=method;
+  fThreshold=threshold;
+  fQuantile=quantile;
+  fCondition=runCond;
+  
+  // parameters are hardcoded for testing purposes so print statements are not necessary at the moment
```

Comment:
is it practical now to cleanup this commented out code?
Do you need it for upcoming development?